packages/email-templates/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0.0",
+  "name": "@unlock-protocol/email-templates",
```

Comment:
You also need to handle this in root package.json since it is required to be built for CI use.

Reply:
added!